### ISSUE TITLE
Terraform Partner Provider Program - Initial Review

### ISSUE THREAD

**Issue by cgriggs01** [KscSDK/terraform-provider-ksyun#54]
I’ve taken a look at the provider here and would like to say great work so far! I do have feedback outlined below that I’d like to see addressed before we move on to the next steps. I’m opening this issues as a sort of checklist for tracking items and discussion. If it works better for you, this changes can be added to a separate branch if any changes conflict with your current build process.  The review was done on the develop branch at git commit `ef31440`

[ ] The provider references a source path of `terraform-provider-ksc` but should be updated to `terraform-provider-ksyun` in the vendor directory [here](https://github.com/KscSDK/terraform-provider-ksyun/blob/master/vendor/vendor.json#L5) .

[ ] We use a [standard GNUmakefile](https://github.com/terraform-providers/terraform-provider-github/blob/master/GNUmakefile) which it looks like you have all but but there are some targets that should be removed. `all`, `dev`, `mac`, `windows`, `linux` because we use a separate build scripts for releasing a new binary of the provider. 

[ ] It’s required that the provider vendor all its dependancies using Go Modules but in addition to that, we ask that you checkin all the required dependancies into a `vendor/` directory. This is so that our nightly testing environment does ping GitHub for ever dependancies in all 100+ TF providers. 

[ ] All official Terraform provider use a [Mozilla Public License 2.0](https://www.mozilla.org/en-US/MPL/2.0/) within each repo, can you add this [LICENSE](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/LICENSE) file to this repository. 

[ ] The [ksyun.erb](https://github.com/KscSDK/terraform-provider-ksyun/blob/master/website/docs/ksyun.erb) file needs to be moved up one directory in the website directory. 

[ ] There are numerous resource built into the provider that don’t have any corresponding documentation files. You will need website docs for each resource and data source. 

[ ] Remove the `bin/` and its contents since the future binaries will be hosted at releases.hashicorp.com

[ ] The documentation is getting close, but you should follow our new provider documentation guide [here](https://www.terraform.io/docs/registry/providers/docs.html)

Let me know if you have any questions. 
Best,
Chris

**Comment by zhuxuanxuan23** [comment]
I modified the project based on your feedback. The result of the modification is on the  branch of [master-verify](https://github.com/KscSDK/terraform-provider-ksyun/tree/master-verify). 
Excuse me, do you have anything to fix? If there are no issues, I will merge to the master branch. Looking forward to your suggestions.
Best,
zhuxuanxuan

**Comment by cgriggs01** [comment]
Yes, please merge into master and we will begin the second review after that. 

Thanks
Chris
